Source/WebCore/page/Page.h

```diff
@@ -33,6 +33,7 @@
 #include "LengthBox.h"
 #include "LoadSchedulingMode.h"
 #include "LocalFrame.h"
+#include "LoginStatus.h"
```

Comment:
Can you forward declare LoginStatus and move this include to the cpp file? We try to avoid adding includes in headers to reduce build times.

Reply:
Actually, I'm wrong about this - you can't forward declare here.

Reply:
Actually, I'm wrong about this - you can't forward declare here. Sorry about that.